### DISCUSSION TITLE
Is there a schema for GitHub actions?

### DISCUSSION THREAD

**Discussion by verdverm** [cue-lang/cue#1171] [Q&A]
Given several projects are generating their GitHub Actions file from CUE, are there any schemas out there that could be imported as a module?

**Comment by verdverm** [comment]
Put this together: https://github.com/hofstadter-io/ghacue/blob/main/schema.cue

**Comment by myitcv** [comment]
Yes, you can import the schemas from https://www.schemastore.org/json/. We do this in the [CUE project itself](https://github.com/cue-lang/cue/blob/0f53054da7243c7a312cd0ad54a1083d67c185d3/internal/ci/vendor/vendor_tool.cue#L46).
